testdata/machineSync.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"machine/sync"
+	"time"
+)
+
+var cond sync.Cond
+
+func delayedResume() {
+	time.Sleep(time.Second)
+	println("notifying")
+	cond.Notify()
+}
+
+func main() {
+	go delayedResume()
+	cond.Wait()
+	cond.Wait()
```

Comment:
Why 2 calls to `cond.Wait()` here?

Reply:
To test that subsequent calls will not block until the condition is cleared.